docs/Playing/index.rst

```diff
@@ -6,6 +6,7 @@ Freeciv21 and any of the varying rulesets that the Longturn.net community likes
 
 
 .. toctree::
+  LT-Guide/index.rst
   faq.rst
   how-to-play.rst
```

Comment:
In the LT Guide there is a note pointing to the How To Play, as a more general guide. I think we should add a corresponding note to the How To Play, pointing to the LT guide, as a guide more focused on Longturn games.

Reply:
Good call out. Thanks